src/components/Calculator.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import calculate from '../logic/calculate';
+
+// eslint-disable-next-line react/prefer-stateless-function
+function Calculator() {
```

Comment:
## [OPTIONAL]
- [ ] Kindly i think that using arrow function would sound good here as we are trying to use ES6